docs/02.API-REFERENCE.md

```diff
@@ -30,6 +30,56 @@ Enum that contains JerryScript API value types:
 
 *New in version 2.0*.
 
+## jerry_object_type_t
+
+Enum that contains JerryScript **object** value types:
+
+ - JERRY_OBJECT_TYPE_NONE - Non object type
```

Comment:
I think JERRY_OBJECT_TYPE_NOT_OBJECT describes this case better.

Reply:
`_NONE` suffixes are used everywhere else in the API to represent invalid enum members, so I'd keep as it to be consistent.